docs/developer/windows/certificate_store/README.md

```diff
@@ -0,0 +1,143 @@
+# Guide to setting up a Windows Server for Certificate Testing
+
+This guide is used to set up a Windows Server for Windows Store Certificate Testing. This guide assumes you have downloaded a Windows 2022 Server ISO and have installed the Windows Server onto a virtual machine. Certificate Templates can only be installed on Windows Server and it must enabled as an Enterprise Certificate Authority. This guide is NOT meant to be a guide on how to set up a Windows Server for production use, and is meant only for setting up an environment to test the Grafana Agent and certificate store.
+
+## Prerequisites
+
+* The install should be fresh with no server roles defined or installed. 
+* You should be logged in via an administrator account.
+
+## Set up as domain controller
+
+1. Add `Active Directory Domain Services`.
+2. Add to a new forest, and when asked for a name, use `test.example.com`.
+3. Add a NETBIOS name `TESTCERT`.
+4. Install.
```

Comment:
Maybe it's just me, but I'm not sure what this is referring to?

Reply:
Added a bit more verbiage, but basically click through til done